lib/kotlin/cross-test-server/build.gradle.kts

```diff
@@ -48,12 +48,6 @@ dependencies {
     testImplementation("org.jetbrains.kotlin:kotlin-test-junit")
 }
 
-kotlin {
-    jvmToolchain {
-        (this as JavaToolchainSpec).languageVersion.set(JavaLanguageVersion.of(8))
-    }
-}
-
```

Comment:
Are the cross-test-client and cross-test-server build.gradle.kts files inheriting from config from lib/kotlin/build.gradle.kts ? It's not obvious to me why nothing is needed in these files.

Reply:
It's not, so removing the `toolChain` here results in using whatever JDK is configured on the host system.
Running the cross-test tools with the default JDK configured on the system is perhaps not a bad thing? Since this results in behavior similar to a real world user of the kotlin lib and they might also use any JDK >= 8.

If you prefer this to also have a `-release` target of 8 or perhaps make server 8 and client 17 let me know :bow:

Reply:
The change is fine with me. I think it's good to explicit whenever doing compilation, whether compiling test code or not. But, I don't think it makes any substantive difference here. So, if omitting it here doesn't cause any problems, it is fine with me.